kratos/processes/vtk_output_process.h

```diff
@@ -129,45 +129,45 @@ class KRATOS_API(KRATOS_CORE) VtkOutputProcess : public Process
     ///@}
     ///@name Operators
     ///@{
-    void CreateMapFromKratosIdToVTKId(ModelPart &model_part);
+    void CreateMapFromKratosIdToVTKId(ModelPart &rModelPart);
```

Comment:
Some documentantion of the methods please

Reply:
Coming up in next commits !